openc3-cosmos-cmd-tlm-api/app/controllers/metadata_controller.rb

```diff
@@ -251,6 +252,13 @@ def action
                type: e.class,
              },
              status: 400
+    rescue OpenC3::SortedOverlapError => e
+      render json: {
+                status: 'error',
+                message: e.message,
+                type: e.class,
+              },
+              status: 409 # Conflict
```

Comment:
Should this be 409.  We're using that for Hazardous Error and probably should be exclusive.